modules/grub/nixos.nix

```diff
@@ -28,10 +24,10 @@ let
         )
 
         # Convert to .pf2
-        ${pkgs.grub2}/bin/grub-mkfont $font --output $out --size ${toString sizes.applications}
+        ${pkgs.grub2}/bin/grub-mkfont $font --output $out --size ${toString fonts.sizes.applications}
       '';
 
-  inherit (config.stylix) imageScalingMode;
+  inherit (config.stylix) imageScalingMode fonts;
```

Comment:
Consider moving this above `mkGrubFont` so it's clearer where `fonts` is coming from.